python/torch_mlir_e2e_test/test_suite/__init__.py

```diff
@@ -15,6 +15,7 @@
     "ConvolutionModule1D_basic",
     "MaxPool2dWith3dInputModule_basic",
     "MaxPool2dWithIndicesWith3dInputModule_basic",
+    "Dlrm_basic",
```

Comment:
We don't want to add this here since the tests in the heavydep tests are manual and don't trigger CI.

Reply:
I see, where should it be called then?
Also, this fails when lowering to TORCH, ill make a test for that but where should that be called?

Reply:
should I do an import in main.py in the folder torchscript_e2e_heavydep_tests ?
from . import facebook_dlrm

Reply:
I don't think we have a good solution for this currently :/

Reply:
i see, I removed it from this file for now.